src/Helpers/Utils.php

```diff
@@ -87,6 +89,26 @@ public function errorResponseIfNotUserHasLevel($currentStatus, &$response, $user
         return false;
     }
 
+    public function errorResponseIfNotOwner($currentStatus, &$response, $user, $asset_id, $message = 'You are not authorized to do this')
+    {
+        if($user === false || $currentStatus) return true;
```

Comment:
This does not follow the style of the other functions around; `return true;` is not inlined on them (nonblocking).